wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -632,118 +600,43 @@ <h3>Loading mask</h3>
-                <li>{% icon 'arrow-down-big' %} arrow-down-big</li>
-                <li>{% icon 'group' %} group</li>
-                <li>{% icon 'media' %} media</li>
-                <li>{% icon 'horizontalrule' %} horizontalrule</li>
-                <li>{% icon 'password' %} password</li>
-                <li>{% icon 'download' %} download</li>
-                <li>{% icon 'duplicate' %} duplicate</li>
-                <li>{% icon 'download-alt' %} download-alt</li>
-                <li>{% icon 'upload' %} upload</li>
-                <li>{% icon 'order' %} order</li>
-                <li>{% icon 'grip' %} grip</li>
-                <li>{% icon 'home' %} home</li>
-                <li>{% icon 'order-down' %} order-down</li>
-                <li>{% icon 'order-up' %} order-up</li>
-                <li>{% icon 'bin' %} bin</li>
-
-                <li>{% icon 'spinner' 'icon spinning' %} spinner</li>
-
-                <li>{% icon 'pick' %} pick</li>
-                <li>{% icon 'redirect' %} redirect</li>
-                <li>{% icon 'view' %} view</li>
-                <li>{% icon 'no-view' %} no-view</li>
-                <li>{% icon 'collapse-up' %} collapse-up</li>
-                <li>{% icon 'collapse-down' %} collapse-down</li>
-                <li>{% icon 'comment' %} comment</li>
-                <li>{% icon 'help' %} help</li>
-                <li>{% icon 'warning' %} warning</li>
-                <li>{% icon 'error' %} error</li>
-                <li>{% icon 'success' %} success</li>
-                <li>{% icon 'date' %} date</li>
-                <li>{% icon 'time' %} time</li>
-                <li>{% icon 'form' %} form</li>
-                <li>{% icon 'site' %} site</li>
-                <li>{% icon 'placeholder' %} placeholder</li>
-                <li>{% icon 'pilcrow' %} pilcrow</li>
-                <li>{% icon 'title' %} title</li>
-                <li>{% icon 'code' %} code</li>
-                <li>{% icon 'openquote' %} openquote</li>
-                <li>{% icon 'chain-broken' %} chain-broken</li>
-                <li>{% icon 'table' %} table</li>
-                <li>{% icon 'login' %} login</li>
-                <li>{% icon 'logout' %} logout</li>
-
-                <li>{% icon 'thumbtack' %} thumbtack</li>
-                <li>{% icon 'resubmit' %} resubmit</li>
-                <li>{% icon 'uni52' %} uni52</li>
-            </ul>
-        </section>
+        {% endpanel %}
+
+        {% panel id="icons" heading="Icons" %}
+
+            {% for source, icons in all_icons %}
+                {% if source == "wagtailadmin/icons" %}
+                    {# Bespoke copy script with good line breaks to review icon update diffs in git. #}
+                    <!-- Copy this section to update the `icons.md` documentation (wagtail_icons.html file). -->
+                    <!-- In DevTools: copy($$('[data-icons-table^="wagtailadmin"]')[0].innerHTML.replace(/\s+/g, ' ').replace(/<t/g, '\n<t')) -->
```

Comment:
Cool! Though we'll need to change this when we fully remove JQuery 😜

Reply:
`$$` is a DevTools feature, equivalent to `Array.from(document.querySelectorAll(`: https://developer.chrome.com/docs/devtools/console/utilities/.

Reply:
Oooo, TIL!